packages/frontpage/lib/schema.ts

```diff
@@ -46,7 +46,7 @@ export const Post = sqliteTable(
   {
     id: integer("id").primaryKey(),
     rkey: text("rkey").notNull(),
-    cid: text("cid").notNull().unique(),
+    cid: text("cid").notNull().default(""),
```

Comment:
Can we not remove the non null constraint here?

Reply:
there's a hard dependency on cids for strong refs currently and if we wanna remove this and make it actually nullable it should be done in a future PR I think